modules/machineset-creating.adoc

```diff
@@ -33,17 +33,38 @@ endif::[]
 [id="machineset-creating_{context}"]
 = Creating a compute machine set
 
-In addition to the ones created by the installation program, you can create your own compute machine sets to dynamically manage the machine compute resources for specific workloads of your choice.
+In addition to the compute machine sets created by the installation program, you can create your own  to dynamically manage the machine compute resources for specific workloads of your choice.
+
+ifdef::vsphere[]
+[NOTE]
+====
+Clusters that are installed with user-provisioned infrastructure have a different networking stack than clusters with infrastructure that is provisioned by the installation program. As a result of this difference, automatic load balancer management is unsupported on clusters that have user-provisioned infrastructure. Compute machine sets cannot add ingress or DNS compute machines to a cluster with user-provisioned infrastructure.
```

Comment:
I don't quite understand this line `Compute machine sets cannot add ingress or DNS compute machines to a cluster with user-provisioned infrastructure`.

Reply:
actually, I think this is covered by the note I have on the <role> param [below](https://github.com/openshift/openshift-docs/pull/57600/files#diff-980b4384fd5a807d47a4c684ef0ea9ffc5018767fc4766cd8ef57adef6af1073R142-R145):
> For clusters that have user-provisioned infrastructure, a compute machine set can only create `worker` and `infra` type machines.

So I could rephrase to match that (listing what is supported instead of what isn't)